cfme/configure/configuration/__init__.py

```diff
@@ -396,6 +270,30 @@ def edit_tag(cat_name, tag_name):
 )
 
 
+class Server(Navigatable, sentaku.Element):
+    def __init__(self, appliance=None, zone=None, name="EVM", sid=1):
+        Navigatable.__init__(self, appliance=appliance)
+        self.zone = zone
+        self.name = name
+        self.sid = sid
+        self.zone.servers.append(self)
+
+
+class Zone(Navigatable, sentaku.Element):
+    def __init__(self, appliance=None, region=None):
+        self.servers = []
+        self.region = region
+        self.name = "default"
+        self.description = "Default Zone"
+        self.region.zones.append(self)
```

Comment:
I don't think the code should assume that it is not in the region's zones, there should be some check ...